RecoTracker/LSTCore/standalone/code/core/lstMath.h

```diff
@@ -1,10 +1,10 @@
-#ifndef SDLMath_h
-#define SDLMath_h
+#ifndef lstMath_h
+#define lstMath_h
 #include <tuple>
 #include <vector>
 #include <cmath>
 
-namespace SDLMath {
+namespace lstMath {
```

Comment:
I think the lower case style applies to all characters 
```suggestion
namespace lstmath {
```
`lst_math` should also be OK